mitosheet/package.json

```diff
@@ -68,10 +68,7 @@
     "lint": "eslint src/ --ext .ts,.tsx --fix",
 
     "prepare": "jlpm run clean && jlpm run build:all",
-    "prepack": "npm run build:all",
-
-    "build:docs:update_frontend": "python docs/make_function_docs.py update_frontend",
```

Comment:
This was somehow duplicated in this PR, so just removing one of the duplicated sets...